lib/jsdom/level2/html.js

```diff
@@ -452,6 +490,9 @@ define('HTMLFormElement', {
 define('HTMLLinkElement', {
   tagName: 'LINK',
   proto: {
+    get _accept() {
```

Comment:
Instead of using a underscore-prefixed getter, use a new internal symbol. Add it to https://github.com/tmpvar/jsdom/blob/master/lib/jsdom/living/helpers/internal-constants.js and then do e.g. `[accept]: "text/css,*/*,q=0.1"` after requiring it.